github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml

```diff
@@ -1475,3 +1475,67 @@ periodics:
         resources:
           requests:
             memory: "29Gi"
+- annotations:
+    testgrid-dashboards: kubevirt-periodics
+    testgrid-days-of-results: "60"
+  cluster: prow-workloads
+  cron: 40 7,15,23 * * *
+  decorate: true
+  decoration_config:
+    grace_period: 5m0s
+    timeout: 4h0m0s
+  extra_refs:
+  - base_ref: main
+    org: kubevirt
+    repo: kubevirt
+  labels:
+    ci.kubevirt.io/prometheus: ""
+    preset-bazel-cache: "true"
+    preset-bazel-unnested: "true"
+    preset-dind-enabled: "true"
+    preset-docker-mirror-proxy: "true"
+  name: periodic-kubevirt-e2e-k8s-1.22-sig-performance-realtime
+  reporter_config:
+    slack:
+      job_states_to_report: []
+  spec:
+    containers:
+    - command:
+      - /usr/local/bin/runner.sh
+      - /bin/sh
+      - -c
+      - |
+        # Create k8s cluster
+        make cluster-up
+
+        # Push and deploy kubevirt
+        make cluster-sync
+
+        FUNC_TEST_ARGS="--noColor --seed=42" make realtime-perftest
+      env:
+      - name: KUBEVIRT_PROVIDER
+        value: k8s-1.22
+      - name: KUBEVIRT_STORAGE
+        value: hpp
+      - name: KUBEVIRT_MEMORY_SIZE
+        value: 9G
+      - name: KUBEVIRT_NUM_NODES
+        value: "2"
+      - name: KUBEVIRT_E2E_REALTIME_PERF_TEST
+        value: "true"
+      - name: KUBEVIRT_E2E_CYCLIC_DURATION_IN_SECONDS
+        value: "60"
+      - name: KUBEVIRT_E2E_REALTIME_LATENCY_THRESHOLD
+        value: "40"
+      - name: KUBEVIRT_HUGEPAGES2M_COUNT
+        value: "512"
+      image: quay.io/kubevirtci/bootstrap:v20210906-994b913
+      name: ""
+      resources:
+        requests:
+          cpu: "12"
+          memory: 44Gi
```

Comment:
I think we should adjust the requests, 26Gi of memory (2 nodes * 9 Gi per node + 8 Gi for cluster management). I think we don't need to request cpus, but if we want to match the other perf lane we should request less than 12 (I guess around 8 for 2 nodes + cluster management).

Reply:
Noted. I reduced the memory to 26Gi and cores to 10.